nss-ice-phantoms/BangazonAPI
# ProductTypes Controller and Tests

# Description
Created ProductTypeController and 5 tests.

## Type of change

# Testing Instructions

1. `git fetch --all`
1. `git checkout meg-producttypes`
1. Recreate database using Tables SQL file and re-add data (will send via Slack) 
1. Open VS, run BangazonAPI, and ensure that CRUD functionalities work, using `http://localhost:5000/api/producttype`
1. Run all 5 tests in Visual Studio.
1. Look over ProductType documentation in README (already on master) and make sure it looks correct.

# Checklist:
